federation/pkg/federation-controller/ingress/ingress_controller.go

```diff
@@ -102,10 +103,14 @@ func NewIngressController(client federation_release_1_4.Interface) *IngressContr
 	ic.ingressInformerStore, ic.ingressInformerController = framework.NewInformer(
 		&cache.ListWatch{
 			ListFunc: func(options api.ListOptions) (pkg_runtime.Object, error) {
-				return client.Extensions().Ingresses(api.NamespaceAll).List(options)
+				// TODO: remove this when Reflector takes an interface rather than a particular ListOptions as input parameter.
```

Comment:
ditto. one comment on VersionizeExtensionsV1Beta1ListOptions method should be enough.